### ISSUE TITLE
Filter control select ordering by alphabetic order

### ISSUE THREAD

**Issue by sdespont** [wenzhixin/bootstrap-table#6201]
### Description

It would be great that the options listed in the select box would be sorted alphabetically.

Typical use case for me is a long list of companies and finding them in a non-sorted list is complicated.

The following example depict the problem, the expected behaviour would be the options listed A-T-Z

![image](https://user-images.githubusercontent.com/646632/171881254-88809ae7-a1ea-4b98-8357-51cb22c93683.png)

https://live.bootstrap-table.com/code/sdespont/11690

**Comment by sdespont** [comment]
@djhvscf Do you have a hint helping me to fix that issue? If you cloud pointing the part of code responsible to create the select list, it would be very useful for me and I will create a pull request with the fix.

**Comment by djhvscf** [comment]
Well, we had this option: `sortSelectOptions`. But looks like we removed the logic where that option was taking effect.

**Comment by sdespont** [comment]
@djhvscf Thanks for the answer.

Would it be correct to just add a call to `sortSelectControl (selectControl, 'asc');`  on line 335 after adding options to the select control?

https://github.com/wenzhixin/bootstrap-table/blob/f861c0b7ac4a5f643a37a51396a79716634c96da/src/extensions/filter-control/utils.js#L289

like
```
      // eslint-disable-next-line guard-for-in
      for (const key in uniqueValues) {
        addOptionToSelectControl(selectControl, uniqueValues[key], key, column.filterDefault)
      }

     sortSelectControl (selectControl, 'asc'); // <--- ADDED
    }
  })
}
```

**Comment by djhvscf** [comment]
Yes, that's correct.. the problem with this option is that it is very expensive.. so we may need to sort the data instead of select control options
